modules/nf-core/fastawindows/tests/main.nf.test

```diff
@@ -0,0 +1,34 @@
+
+nextflow_process {
+
+    name "Test Process FASTAWINDOWS"
+    script "../main.nf"
+    process "FASTAWINDOWS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "fastawindows"
+
+    test("test-fastawindows") {
+
+        when {
+            process {
+                """
+                input[0] = [
+				    [ id:'test' ], // meta map
+				    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+				]
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+
```

Comment:
wanna add the test for stub too?

Reply:
Thank you @sateeshperi 

That's done now.